moveit_core/collision_detection/src/world.cpp

```diff
@@ -76,10 +76,10 @@ void World::addToObject(const std::string& id, const std::vector<shapes::ShapeCo
 
   int action = ADD_SHAPE;
 
-  ObjectPtr& obj = objects_[id];
+  ObjectPtr& obj = objects_[object_id];
```

Comment:
I don't have a strong opnion on the `id` to  `object_id` change , but it creates a lot of noise in the diff that doesn't seem related to the purpose of the PR.

Reply:
These types of changes are mostly accumulated comments from the last PR. I agree that it's become pretty big. All the more reason to get it over with, in my opinion.

Reply:
I suggest addressing these variable renamings in another PR.